docs/content/manpages/geowave-recalcstats.txt

```diff
@@ -0,0 +1,26 @@
+//:geowave-recalcstats(1)
+//:================
+//::doctype: manpage
+
+NAME
+//:----
+
+geowave remote recalcstats - Calculate the statistics of an existing GeoWave dataset
+
+SYNOPSIS
+//:--------
+
+*geowave remote recalcstats* [options] <store name> <adapterId> <statId>
```

Comment:
If it's a bug that it shows up, I can remove that reference in the doc and we can file an issue to fix it later on.